### PR TITLE
swift: Backup account/container

### PR DESCRIPTION
**Pull request by paladox**

### CODE REVIEW

File: modules/base/templates/backups/miraheze-backup.py.erb [hunk 1]
```diff
@@ -68,6 +68,17 @@ def backup_sslkeys(dt: str):
     os.remove('sslkeys.tar.gz')
 
 
+def backup_swift_account_container(dt: str):
+    tar = tarfile.open('swift_account_container.tar.gz', 'w:gz')
+    tar.add('/srv/nodes', arcname='swift_account_container')
+    tar.close()
+
+    pca_connection().put('swift_account_container.tar.gz', f'swift_account_container/{dt}.tar.gz', False)
+    pca_web('POST', f'https://storage.bhs.cloud.ovh.net/v1/AUTH_76f9bc606a8044e08db7ebd118f6b19a/sslkeys/{dt}.tar.gz', 4)
```
Comment (The-Voidwalker) [inline modules/base/templates/backups/miraheze-backup.py.erb:77]: probably shouldn't be putting this in the sslkeys dir

Comment (The-Voidwalker) [inline modules/base/templates/backups/miraheze-backup.py.erb]:
```suggestion
    pca_web('POST', f'https://storage.bhs.cloud.ovh.net/v1/AUTH_76f9bc606a8044e08db7ebd118f6b19a/swift_account_container/{dt}.tar.gz', 4)
```

Comment (paladox) [reply modules/base/templates/backups/miraheze-backup.py.erb]: Thanks, done.

Comment (paladox) [reply modules/base/templates/backups/miraheze-backup.py.erb]: Fixed. Thanks!

**Pull request merged**